service.watchedlist/changelog.txt

```diff
@@ -0,0 +1,100 @@
+version 1.3 (24.04.2020)
```

Comment:
it would be nice (not mandatory though) if you could define the `<news>` tag in addon.xml with the most recent changes. Changelog is not parsed by kodi in either way whilst with news users can easy check the changes in the UI